pkg/apis/openshiftcontrollermanager/v1alpha1/types.go

```diff
@@ -39,10 +39,24 @@ type OpenShiftControllerManagerOperatorConfigSpec struct {
 	// observedConfig holds a sparse config that controller has observed from the cluster state.  It exists in spec because
 	// it causes action for the operator
 	ObservedConfig runtime.RawExtension `json:"observedConfig"`
+
+	// additionalTrustedCA references the additional trusted certificate authorities that operator should attempt to configure for the build controller.
+	AdditionalTrustedCA *AdditionalTrustedCA `json:"additionalTrustedCA,omitempty"`
```

Comment:
This breaks the "no pointers" rule for API, but results in much cleaner code.
This will likely go away when certs are split out by hostname.

@deads2k do we need protobuf as well as json?